src/orders/components/OrderDetailsPage/OrderDetailsPage.tsx

```diff
@@ -298,6 +300,36 @@ const OrderDetailsPage = (props: OrderDetailsPageProps) => {
                 </>
               )}
 
+              {order && isOrderUnconfirmed && orderDiscountContext && (
+                <>
+                  <OrderSummary
+                    order={order}
+                    onMarkAsPaid={onMarkAsPaid}
+                    useLegacyPaymentsApi={!orderShouldUseTransactions(order)}
+                    onLegacyPaymentsApiCapture={onPaymentCapture}
+                    onLegacyPaymentsApiRefund={onPaymentRefund}
+                    onLegacyPaymentsApiVoid={onPaymentVoid}
+                    isEditable
+                    onShippingMethodEdit={onShippingMethodEdit}
+                    errors={errors}
+                    {...orderDiscountContext}
```

Comment:
suggestion: why can't we use context in that component? if we're adding it to `OrderDetailsPage` just to spread it here we should probably use context inside `OrderSummary instead